src/Extensions/Microsoft.Diagnostics.Monitoring.Extension.Common/Microsoft.Diagnostics.Monitoring.Extension.Common.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
```suggestion
﻿<Project Sdk="Microsoft.NET.Sdk">
```

Reply:
You might have to pull in other libraries (e.g. Microsoft.Extensions.*) to get this working after changing the SDK.